pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -191,7 +188,12 @@ func Run() {
 		}
 	}
 
-	domainCtx := domainContext{usbAccess: true}
+	// These settings can be overridden by GlobalConfig
+	domainCtx := domainContext{
+		usbAccess:           true,
+		vdiskGCTime:         3600,
+		domainBootRetryTime: 600,
```

Comment:
Why do we need this initialization? Aren't these provided as defaults from GlobalConfig? Why duplicate the Default values?

Reply:
Because it was done in the old code, and I'm not touching how that initialization is done as part of this PR. 
The old code just above was:
var vdiskGCTime = time.Duration(3600) * time.Second        // Unless from GlobalConfig	
var domainBootRetryTime = time.Duration(600) * time.Second // Unless from GlobalConfig